tests/Resources/modules/translationtest/views/templates/admin/index.html.twig

```diff
@@ -4,7 +4,7 @@
 {% block content %}
     <div class="row">
         <div class="col-lg-8">
-            <h2>{{ 'Hello World'|trans({}, 'Modules.Translationtest.Translationtest') }}</h2>
+            <h2>{{ 'Hello World'|trans({}, 'Modules.Translationtest.Translation_test') }}</h2>
```

Comment:
why do we have an underscore here? What is/are the rules to guess which domain name we should use in this case ?

Reply:
Yep, same question.

Reply:
@LouiseBonnard You can ignore this, they are test resources.

@mickaelandrieu It belongs to the legacy fallback system, where you declare a domain and use already existing translations savec in the legacy format. In order for the "use existing translations" part to work, the domain name must follow some very specific rules, but basically: `Modules.<Nameofthemodule>.<Name_of_the_file_without_extension>`. This is so, specially the underscore part, because translations are stored that way in the legacy files.

Rest assured, this is all being documented and will be published soon.